components/repository/repository.html

```diff
@@ -1,4 +1,4 @@
-<div class="repository-view animated fadeInLeft">
+<div class="repository-view animated fadeInLeft" data-bind="style: { tabSize: ungit.config.diffTabSize }">
```

Comment:
```suggestion
<div class="repository-view animated fadeInLeft" data-bind="attr: { style: 'tab-size: ' + ungit.config.diffTabSize }">
```
knockout has a "nice" feature to automatically append `px` to numeric values. But in this case we don't want it, so we need to use `attr` instead.
https://github.com/knockout/knockout/blob/45e86bf0347cf742ca359bab051ea82fdfae7a5d/src/binding/defaultBindings/style.js#L25-L27

Reply:
Is there any plan or willing to move the UI to a more modern framework? (React / Vue.js)